### ISSUE TITLE
Accessibility: Can't access relay settings page.

### ISSUE THREAD

**Issue by shortwavesurfer2009** [variablefate/ridestr#40]
I noticed a bars icon on the ride page but cannot get to it using Talkback. It just immediately goes to the account icon next to it.

<img width="814" height="1808" alt="Image" src="https://github.com/user-attachments/assets/9296be10-8eb5-4e2d-bb97-bce5ac30cd4f" />

**Comment by variablefate** [comment]
I'll knock this out tomorrow. reviewing accessibility stuff right now.

**Comment by variablefate** [comment]
Fixed in 209cdc8. Added TalkBack semantic roles and content descriptions to RelaySignalIndicator and other interactive UI elements. Also fixed the oversized spacing from the initial 48dp touch target approach.
